cdisc_rules_engine/services/data_services/dummy_data_service.py

```diff
@@ -124,7 +124,17 @@ def has_all_files(self, prefix: str, file_names: List[str]) -> bool:
         return True
 
     def join_split_datasets(self, func_to_call: Callable, dataset_names, **kwargs):
```

Comment:
This should probably be called `concat_split_datasets` or `union_split_datasets`. `join` implies something else